tests/auth-util.test.ts

```diff
@@ -51,6 +51,18 @@ test('check if still valid token inside buffer has expired', () => {
   expect(hasExpired).toBe(true)
 })
 
+test('expire time as string gets correctly converted', () => {
+  const expectedEpoch = Math.round(Date.now() / 1000 + 55555)
+  const epochSumCalculated = epochAtSecondsFromNow('55555')
+  expect(expectedEpoch).toBe(epochSumCalculated)
+})
+
+test('expire time as int gets correctly converted', () => {
+  const expectedEpoch = Math.round(Date.now() / 1000 + 55555)
+  const epochSumCalculated = epochAtSecondsFromNow(55555)
+  expect(expectedEpoch).toBe(epochSumCalculated)
+})
```

Comment:
Oo, tiny chance of failing these tests if line 55 and 56 happen on second 0.49999 and second 0.5000, making them round different ways. Still, considering the speed at which these two calls probably happen, the chances of this is probably in a range like 1:1e9 or something, so I think it's fairly safe.

Reply:
I take that risk :stuck_out_tongue: